go/vt/topotools/tablet.go

```diff
@@ -246,3 +246,32 @@ func TabletIdent(tablet *topodatapb.Tablet) string {
 func TargetIdent(target *querypb.Target) string {
 	return fmt.Sprintf("%s/%s (%s)", target.Keyspace, target.Shard, target.TabletType)
 }
+
+// TabletEquality returns true iff two Tablets are identical for testing purposes
+func TabletEquality(left, right *topodatapb.Tablet) bool {
```

Comment:
Why do we need a custom function? Why can't we use `cmp.Comparer`?
https://github.com/golang/protobuf/issues/1238#issuecomment-723547616

Reply:
We don't need the custom function. `proto.Equal` in `cmp.Diff` is working just fine. I have changed the code and removed that function.